src/main/java/com/poiji/bind/Poiji.java

```diff
@@ -370,6 +372,93 @@ private static HSSFPropertyStream deserializerPropertyStream(PoijiExcelType exce
         unmarshaller.unmarshal(type, consumer);
     }
 
+    public static <T> void toExcel(final Sheet sheet,
+                                   final Collection<T> objects,
+                                   final Class<T> type) {
+        toExcel(sheet, objects, type, PoijiOptionsBuilder.settings().build());
+    }
+
+    public static <T> void toExcel(final Sheet sheet,
+                                   final Collection<T> objects,
+                                   final Class<T> type,
+                                   final PoijiOptions options) {
+        Field[] fields = type.getDeclaredFields();
+        List<Field> headers = new ArrayList<>();
+
+        int rowIndex = 0;
+        Row header = sheet.createRow(rowIndex);
+        for (int i = 0; i < fields.length; i++) {
+            Field field = fields[i];
+            field.setAccessible(true);
+            ExcelCellName[] annotations = field.getAnnotationsByType(ExcelCellName.class);
+            for (ExcelCellName annotation : annotations) {
+                headers.add(field);
+                Cell cell = header.createCell(i);
+                cell.setCellValue(annotation.value());
+                break;
+            }
+        }
+
+        for (T dto : objects) {
+            Row dataRow = sheet.createRow(++rowIndex);
+            int cellIndex = 0;
+            for (Field field : headers) {
+                Cell cell = dataRow.createCell(cellIndex++);
+                Object object;
+                try {
+                    object = field.get(dto);
+                } catch (IllegalAccessException e) {
+                    throw new PoijiException("Error during processing POJO to Excel", e);
+                }
+                setCellValue(cell, object, field.getType(), options);
+            }
+        }
+    }
+
+    private static void setCellValue(Cell cell, Object value, Class<?> clazz, PoijiOptions options) {
```

Comment:
I think we need to move this logic to another class, we have a similar class called https://github.com/ozlerhakan/poiji/blob/master/src/main/java/com/poiji/config/DefaultCasting.java for deserialization.